### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by Kdoggg666**
## Description

New code of conduct section in nav and footer

## Pull request type

- [ ] #M01 (Miscellaneous User Story #01)
- [ ] #P02 (Participant User Story #02)
- [ ] #S03 (Staff User Story #03)
- [ ] #A04 (Admin User Story #04)
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change

- [ ] Yes
- [] No

### CODE REVIEW

File: resources/urls.py [hunk 1]
```diff
@@ -7,4 +7,5 @@
     path('delete/<int:resource_id>/', views.delete_resource,
          name='delete_resource'),
     path('edit/<int:resource_id>/', views.edit_resource, name='edit_resource'),
+
```
Comment (stefdworschak) [inline resources/urls.py:10]: Do we need this extra line?

File: templates/code-of-conduct.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+            </ol>
+            <p>By participating in a Code Institute Hackathon, you agree to abide by this code of conduct and any decisions made by Code Institute regarding violations of it.</p>
+    </div>
+</div>
+
+
+
+
+{% endblock %}
```
Comment (stefdworschak) [inline templates/code-of-conduct.html:34]: Could you please add an extra line at EOF?

**Review by stefdworschak — APPROVED** [review]
LGTM

File: home/views.py [hunk 3]
```diff
@@ -146,3 +146,9 @@ def test_404(request):
     response = render(request, '404.html')
     response.status_code = 404
     return response
+
+def codeofconduct(request):
+    """
+        A view to show the hackathon code of conduct.
+    """
+    return render(request, 'code-of-conduct.html')
```
[the remark this answers is not in the document]
Comment (Kdoggg666) [reply home/views.py]: Thanks Stefan, 

I have resolved these in commit number 3774100.

File: resources/urls.py (hunk 1 above)
Comment (Kdoggg666) [reply resources/urls.py]: resolved in 3774100

File: templates/code-of-conduct.html (hunk 2 above)
Comment (Kdoggg666) [reply templates/code-of-conduct.html]: resolved in 3774100

**Pull request merged**